Guideline.md

```diff
@@ -382,7 +382,34 @@ Speech recognition accuracy can be as high as 90% when your video has one person
 **Provide sufficient time between talks, events etc.** 
 Remember it can take more time to navigate a venue with a mobility device, especially if it is crowded. So consider the suitable time between talks/events for your conference. 
 
-## 15. Sources
+## 15. Financial considerations
+
+Some of the things suggested here cost money: some more, some less. These costs may seem prohibitively high given the tight budget of conferences.
+
+Low conference ticket prices are an important factor to enable inclusion, so simply increasing ticket prices to enable a child day care may not be in the best interests of inclusion efforts. 
+
+Also note once again, it's ok to initially focus on the topics that do not cost money and then later take the next steps.
+
+Here are some concrete ideas to finance diversity and inclusion efforts at your conference:
+
+* Bring costs down by asking for volunteers
+    * Maybe someone knows sign language in your community?
+    * Maybe a significant other who is not in IT is willing to help out in the nursery to assist a professional carer? 
+* Set up specific sponsor packages
+* Allow attendees to optionally donate when buying their ticket
+    * Potentially give them a small token of appreciation like a button, a free drink ticket at the conference party ...
+* Set up an (all year) sponsor fund to collect donations
+    * [opencollective.com](https://opencollective.com)
+    * [sfconservancy.org](https://sfconservancy.org)
+
+A slightly more out of the box approach could be to save money that usually goes towards speakers. Some of the suggestions can have the bonus effect at getting more people involved in speaking in public which is also great for diversity. Also keep in mind that while conferences are traditionally very focused on having speakers share their knowledge, a very significant (if not even the most significant) aspect of conferences is networking.
+
+* Focus more on regional speakers to reduce travel costs
+* Allow for more user generated content by providing an unconference track with lightning talks (which can even just be about “what do I do at work”)
```

Comment:
I've encountered `lightning talks` to be talks promoting products and doing advertisement.. not sure it's a good word here? How about `un-conference track with talks which can even just be about “what do I do at work”.`

Reply:
yeah .. sometimes the term "lightning talk" is "abused" to simply give all the sponsors a slot to promote their products :-/

Reply:
I have incorporated your suggestion.